ClientLib/src/main/java/com/o3dr/services/android/lib/coordinate/Frame.java

```diff
@@ -0,0 +1,66 @@
+package com.o3dr.services.android.lib.coordinate;
+
+import com.MAVLink.enums.MAV_FRAME;
+
+/** TODO This is the start of using a more generic location/frame class
+ *  The frame for LatLongAlt would only ever be GLOBAL and not LOCAL.
+ */
+
+public enum Frame {
+    GLOBAL_ABS      ("Absolute" , "Abs", 0 ),   // Absolute means Above Mean Sea Level AMSL
+    LOCAL_NED       ("Local NED", "NED", 1 ),
+    MISSION         ("Mission"  , "MIS", 2 ),
+    GLOBAL_RELATIVE ("Relative" , "Rel", 3 ),   // Relative to HOME location
+    LOCAL_ENU       ("Local ENU", "ENU", 4 ),
+    GLOBAL_TERRAIN  ("Terrain"  , "Ter", 10);   // Relative to Terrain Level. (Either measured or from STRM)
+
+    private final int frame;
+    private final String name;
+    private final String abbreviation;
```

Comment:
@billbonney I'd advocate for the removal of the `frame` field since it can be substitute by the enum itself.